userspace/otpilot/src/main.rs

```diff
@@ -169,10 +161,11 @@ fn run() -> TockResult<()> {
 
     //////////////////////////////////////////////////////////////////////////////
 
-    let mut flash_read: [u8; flash::MAX_BUFFER_LENGTH] = [0; flash::MAX_BUFFER_LENGTH];
-    writeln!(console, "reading flash ...")?;
-    flash::get().read(0x5000, &mut flash_read, 8)?;
-    writeln!(console, "flash_read = {:?}", &flash_read[0..8])?;
+    {
+        let mut flash_read: [u8; flash::MAX_BUFFER_LENGTH] = [0; flash::MAX_BUFFER_LENGTH];
+        flash::get().read(0x5000, &mut flash_read, 8)?;
```

Comment:
Is `0x5000` a magic number?  Given that this is inside a "demo" function, do we care?

Reply:
Given that this function is now gone, never mind.